huxley/www/js/components/ChairAttendanceView.js

```diff
@@ -42,6 +42,7 @@ var ChairAttendanceView = React.createClass({
 
   componentDidMount() {
     var user = CurrentUserStore.getCurrentUser();
+    this._mapAssignments();
     this._delegatesToken = DelegateStore.addListener(() => {
```

Comment:
This is a bug I noticed and decided to wrap it into this PR since I was making changes anyways